frontend/.eslintrc

```diff
@@ -326,6 +326,10 @@
           {
             "selector": "Literal[value=/\\bOpen Data Hub\\b/i],JSXText[value=/\\bOpen Data Hub\\b/i]",
             "message": "Do not hard code product name `Open Data Hub`. Use `~/utilities/const#ODH_PRODUCT_NAME` instead."
+          },
+          {
+            "selector": "JSXElement[openingElement.name.name=/Modal/]:has(JSXAttribute[name.name=/(isOpen|open)/][value])",
+            "message": "Do not control modals visibility with 'isOpen|open', use conditional rendering instead."
```

Comment:
This rule could use some fixing. It does not allow any children of the modal to pass `isOpen` with a value to its component (like a `Select` in the body of the modal).